spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/zuul/SimpleZuulServerApplicationTests.java

```diff
@@ -83,13 +90,26 @@ public void getOnSelf() {
 	}
 
 	@Test
-	public void getOnSelfViaFilter() {
+	public void getOnSelfViaFilterShouldSucceed() {
+		getOnSelfViaFilter();
+	}
+
+	@Test
+	public void exceptionOccurredInFilterShouldBeLogged() throws Exception {
```

Comment:
@spencergibb As you can see from this test: exception was thrown in prefilter, but response 200 with empty body was returned to the client. Is it correct behaviour? I suppose no.